src/PropTypeShapes.js

```diff
@@ -0,0 +1,11 @@
+import PropTypes from "prop-types";
```

Comment:
This is a comment on the file name, not this line, but I can't leave a comment on the file name itself.

I'm not actually sure what the most common conventions are here, but I feel that it might be better to give this file a kebab-case name, like `prop-types.js`. The reason behind my thinking is that in most of our files that are `CamelCase.js`, we have a default export of a class whose name matches the file name, so the file name describes the type of thing being exported.

In this file, we don't have a natural default export, so we're only providing named exports. We don't actually have a class named `PropTypeShapes`, so it might be strange to name the file after that.

What do you think?

Reply:
I agree! I'll change the file name to `prop-types.js`.